packages/ui-components/src/components/CheckboxList.js

```diff
@@ -83,55 +86,72 @@ export const CheckboxList = ({ list, title, selectedItems, setSelectedItems }) =
 
   const Title = ({ title, items }) => (
     <FlexStart>
-      <StyledHeader>{title}</StyledHeader>
-      <StyledSubHeader>{`${numberOfChecked(items)}/${items.length} selected`}</StyledSubHeader>
+      <FormControl>
+        <FormControlLabel
+          label={title}
+          control={
+            <Checkbox
+              onClick={handleCheckAll(enabledItems)}
+              checked={
+                numberOfChecked(enabledItems) === enabledItems.length && enabledItems.length !== 0
+              }
+              indeterminate={
+                numberOfChecked(enabledItems) !== enabledItems.length &&
+                numberOfChecked(enabledItems) !== 0
+              }
+              disabled={enabledItems.length === 0}
+              inputProps={{ 'aria-describedby': `subtitle-select-all-${title}` }}
+            />
+          }
+        />
+        <StyledSubHeader id={`subtitle-select-all-${title}`}>{`${numberOfChecked(items)}/${
+          items.length
+        } selected`}</StyledSubHeader>
+      </FormControl>
     </FlexStart>
   );
 
-  const customList = (title, items) => (
+  // If the list item has a tooltip, wrap it in a tooltip, otherwise just return the list item
+  const CheckboxWrapper = ({ tooltip, children }) =>
+    tooltip ? (
+      <Tooltip describeChild title={tooltip} placement="bottom">
+        {children}
+      </Tooltip>
+    ) : (
+      <div>{children}</div>
+    );
+
+  return (
     <StyledCard>
-      <StyledCardHeader
-        avatar={
-          <Checkbox
-            onClick={handleCheckAll(items)}
-            checked={numberOfChecked(items) === items.length && items.length !== 0}
-            indeterminate={numberOfChecked(items) !== items.length && numberOfChecked(items) !== 0}
-            disabled={items.length === 0}
-            inputProps={{ 'aria-label': 'all items selected' }}
-          />
-        }
-        title={<Title title={title} items={items} />}
-      />
+      <StyledCardHeader title={<Title title={title} items={list} />} />
       <Divider />
       <List dense component="div" role="list">
-        {items.map(item => {
-          const { name, code } = item;
-          const labelId = `transfer-list-all-item-${name}-label`;
-
+        {list.map(item => {
+          const { name, code, disabled, tooltip } = item;
           return (
-            <ListItem key={code} role="listitem" button onClick={handleCheck(item)}>
-              <ListItemIcon>
-                <Checkbox
-                  checked={
-                    selectedItems.findIndex(selectedItem => selectedItem.code === code) !== -1
-                  }
-                  tabIndex={-1}
-                  disableRipple
-                  inputProps={{ 'aria-labelledby': labelId }}
-                />
-              </ListItemIcon>
-              <ListItemText id={labelId} primary={name} />
+            <ListItem key={code} role="listitem">
```

Comment:
Any reason that you didn't include the white bottom border on the list items?

Reply:
Ah that would because I actually forgot. Will add that!